src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private final int CAPACITY = 12;
+    private final int EMPTY = -1;
 
-    public int total = -1;
+    private int[] numbers = new int[CAPACITY];
+
+    public int total = EMPTY;
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == EMPTY;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == CAPACITY-1;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return EMPTY;
         return numbers[total];
     }
 
     public int countOut() {
         if (callCheck())
-            return -1;
+            return EMPTY;
```

Comment:
W klasie DefaultCountingOutRhymer stałe o wartości -1 odnoszące się do pola total (też w metodzie callCheck), to nie jest ta sama stała o wartości -1 oznaczające domyślną zwracaną przez wyliczankę wartość (w metodach peekaboo i countOut). Powinny to być osobne stałe.